modules/tests/shared/src/test/scala/lucuma/core/math/parser/AngleParsersSuite.scala

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.math.parser
+
+import cats.kernel.laws.discipline.*
+import lucuma.core.math.Declination
+import lucuma.core.math.arb.ArbAngle
+import lucuma.core.math.arb.ArbRightAscension.*
+import lucuma.core.optics.laws.discipline.FormatTests
+import munit.*
+
+class AngleParsersSuite extends DisciplineSuite with AngleParsers:
```

Comment:
Maybe we can test that the returned values are the expected ones?

Reply:
I updated the tests to do actual comparisons